airflow/www/templates/airflow/stats.html

```diff
@@ -0,0 +1,97 @@
+{#
+ Licensed to the Apache Software Foundation (ASF) under one
+ or more contributor license agreements.  See the NOTICE file
+ distributed with this work for additional information
+ regarding copyright ownership.  The ASF licenses this file
+ to you under the Apache License, Version 2.0 (the
+ "License"); you may not use this file except in compliance
+ with the License.  You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing,
+ software distributed under the License is distributed on an
+ "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ KIND, either express or implied.  See the License for the
+ specific language governing permissions and limitations
+ under the License.
+#}
+
+{% extends base_template %}
+
+{% block content %}
+  <div>
+    <h2>Airflow stats</h2>
+    <p>If you are interested in more detailed statistics please consider using
+      <a href="https://airflow.apache.org/docs/stable/metrics.html" target="_blank">statsd integration</a>.
+    </p>
+    <h3>Task instances</h3>
+     <table id="tis-table" class="table table-bordered table-striped" style="width: 50%">
```

Comment:
```suggestion
     <table class="table table-bordered table-striped" style="width: auto; min-width: 50%;">
```
The id isn't necessary
Probably not a big deal with this limited content, but setting the `width: 50%` can yield some undesirable effects with narrower screen-widths. This `width/min-width` combo will be a bit more flexible.